src/js_object_accessor.hpp

```diff
@@ -233,8 +234,8 @@ class NativeAccessor {
         return false;
     }
 
-    bool is_same_set(realm::object_store::Set const& set, ValueType const& value) const {
-        throw std::runtime_error("'Set' type support is not implemented yet");
+    bool is_same_set(realm::object_store::Set const &set, ValueType const &value) const {
+        return false;
```

Comment:
Is it not possible to return true here?

Reply:
Well, that depends on the semantics we are going for.  It should always be `false` for different sets -- even if they contain the same elements -- because they create different objects in Core.
We could return `true` if a Set object is compared to itself, I guess.

Reply:
We could compare the (table key, object key) pair, right?

Reply:
Yes.  I've changed `is_same_set` to call the `==` overload operator from Object Store's Set, which does exactly that.